packages/ddl-shim/src/upgrade.ts

```diff
@@ -433,6 +439,16 @@ class DDLUpgrade {
         }
     }
 
+    rule2condition(_: DDL1.IFilterRule): DDL2.IMappingConditionType {
```

Comment:
Maybe this should be more sensibly named?

Reply:
"filterRule2MappingConditionType"?